vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/AudioMessagePlaybackTracker.kt

```diff
@@ -110,21 +111,23 @@ class AudioMessagePlaybackTracker @Inject constructor() {
 
     fun getPlaybackState(id: String) = states[id]
 
-    fun getPlaybackTime(id: String): Int {
+    fun getPlaybackTime(id: String): Int? {
```

Comment:
Why those changes?

Reply:
For Voice Broadcast, I need to distinguish a paused playback at 0sec from another which has no saved position, so I decided to return null in that case. The first time we play a voice broadcast which is live, the user should automatically jump to the last chunk, if he decided to manually scroll in the seekBar, the playback will start from that position (eg. 0 sec)

Reply:
Ok, maybe a sealed class would have been cleaner, but ok!

Reply:
I reused this object, I agree there are several things to improve: having the recording in the states list is a bit weird, same about having a percentage but not the duration, and like you said we don't really need this method and could have it in the sealed class (which already exists)